frontend/src/Episode/History/EpisodeHistoryRow.js

```diff
@@ -102,14 +101,28 @@ class EpisodeHistoryRow extends Component {
           />
         </TableRowCell>
 
+        <TableRowCell>
+          <EpisodeFormats formats={customFormats} />
+        </TableRowCell>
+
+        <TableRowCell>
+          <Tooltip
```

Comment:
Doesn't seem super useful to have a tooltip with the custom formats when they're always shown in the column to the left.

Reply:
would make sense to me to remove the tooltip indeed!